samples/bluetooth/iso_broadcast/src/main.c

```diff
@@ -43,41 +44,44 @@ static struct bt_iso_chan_ops iso_ops = {
 	.disconnected	= iso_disconnected,
 };
 
-static struct bt_iso_chan_io_qos iso_tx_qos = {
-	.sdu = sizeof(uint32_t), /* bytes */
-	.rtn = 2,
-	.phy = BT_GAP_LE_PHY_2M,
+static struct bt_iso_chan_io_qos iso_tx_qos[] = {
+	{ .sdu = sizeof(uint32_t), /* bytes */
+	  .rtn = 1,
+	  .phy = BT_GAP_LE_PHY_2M, },
 };
 
-static struct bt_iso_chan_qos bis_iso_qos = {
-	.tx = &iso_tx_qos,
+static struct bt_iso_chan_qos bis_iso_qos[] = {
+	{ .tx = &iso_tx_qos[0], },
```

Comment:
Doesn't seem like we need to make these arrays

Reply:
Fixed.